src/Ledger/PParams.lagda

```diff
@@ -32,20 +32,20 @@ data PParamGroup : Set where
 
 record PParams : Set where
   field -- Network group
-        maxBlockSize      : ℕ
-        maxTxSize         : ℕ
-        maxHeaderSize     : ℕ
-        maxValSize        : ℕ
-        pv                : ProtVer -- retired, keep for now
+        maxBlockSize     : ℕ
```

Comment:
(same comment as above)
This change was because this is a .agda file, not a .lagda file and I think @WhatisRT prefers to not have extra indents in .agda files (although think it would be simpler, and would prefer, if we could use one indentation style throughout---namely, a style that looks good in both emacs and the generated pdf).

Reply:
More cleanup undoing: note that this is a literate agda file.